play-interop-test-scala/src/test/scala/akka/grpc/scalatestplus/play/ServerGrpcClient.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.scalatestplus.play
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.grpc.internal.{ AkkaGrpcClient, AkkaGrpcClientFactory }
+import akka.stream.Materializer
+import com.typesafe.config.{ Config, ConfigFactory, ConfigValueFactory }
+import org.scalatestplus.play.NewServerProvider
+import play.api.test.ServerEndpoint
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ Await, ExecutionContext }
+
+/**
+ * Helpers to test gRPC clients with Play using ScalaTest.
+ */
+trait ServerGrpcClient { this: NewServerProvider =>
```

Comment:
The source code for this ScalaTest class is actually exactly the same as the source code for the specs2 class. Whether or not we can create a common bit of code depends on how the code is eventually split between the akka-grpc, play-specs2 and scalatestplus-play projects.